ffi/src/schema.rs

```diff
@@ -52,6 +53,7 @@ pub struct EngineSchemaVisitor {
         data: *mut c_void,
         sibling_list_id: usize,
         name: KernelStringSlice,
+        is_nullable: bool,
         contains_null: bool, // if this array can contain null values
```

Comment:
```suggestion
        is_nullable: bool,
```
(see commentary about the TODO flagged below)